app/scripts/controllers/pod.js

```diff
@@ -87,6 +87,49 @@ angular.module('openshiftConsole')
       }
     };
 
+    /**
+     *  Will set the newTerm's isActive/isUsed to true, while deactivating the old's
+     */
+    $scope.onTerminalSelectChange = function(newTerm) {
+      // Deactivate all terminals
```

Comment:
But the code below sets `isActive` to false for everything except the last selected.